tailwind.config.js

```diff
@@ -67,6 +67,9 @@ export default {
             colors: {
                 gray: require('tailwindcss/colors').zinc,
             },
+            fontSize: {
+                'xxs': ['.5rem','.75rem'],
+            },
```

Comment:
font-size and line-height are separate classes in Tailwind so I would prefer we keep them separate in any custom classes we created.

There already exists a `.75rem` line-height class `leading-3`. So let's use that, remove the line height from this new `xxs` class. Then this should be good.

Reply:
Hi @Orrison 
Changes done as per your suggestion. I don't know about that leading-3 class. sorry for that.

Reply:
No problem at all!